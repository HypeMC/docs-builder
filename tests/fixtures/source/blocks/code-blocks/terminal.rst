
.. code-block:: terminal
    git clone git@github.com:symfony/symfony.git

.. code-block:: terminal

    $ cowsay 'eat more chicken'
