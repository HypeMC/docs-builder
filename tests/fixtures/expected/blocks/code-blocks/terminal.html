<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />

            
    </head>

    <body>
            <div translate="no" class="notranslate codeblock codeblock-loc-1 codeblock-terminal codeblock-bash">
    <div class="codeblock-scroll">
        <pre class="codeblock-lines">1</pre>
        <pre class="codeblock-code"><code>git <span class="hljs-built_in">clone</span> git@github.com:symfony/symfony.git
</code></pre>
    </div>
</div>

<div translate="no" class="notranslate codeblock codeblock-loc-1 codeblock-terminal codeblock-bash">
   <div class="codeblock-scroll">
       <pre class="codeblock-lines">1
2</pre>
       <pre class="codeblock-code">
           <code>
               <span class="hljs-prompt">$</span> cowsay
               <span class="hljs-string">'eat more chicken'</span>
               <span class="hljs-prompt">$</span> cowsay
               <span class="hljs-string">'mmmm'</span>
           </code>
       </pre>
   </div>
</div>

<div translate="no" class="notranslate codeblock codeblock-loc-1 codeblock-terminal codeblock-bash">
   <div class="codeblock-scroll">
       <pre class="codeblock-lines">1
2
3</pre>
       <pre class="codeblock-code">
           <code>
               <span class="hljs-prompt">C:\&gt;</span> CIV
               <span class="hljs-comment"># Civilization for DOS - my first computer game!</span>
           </code>
       </pre>
   </div>
</div>

    </body>
</html>
